packages/platform-browser/test/browser/bootstrap_spec.ts

```
@@ -449,7 +449,9 @@ function bootstrap(
                return compiler.compileModuleAsync(AsyncModule).then(factory => {
                  expect(() => factory.create(ref.injector))
                      .toThrowError(
-                         `BrowserModule has already been loaded. If you need access to common directives such as NgIf and NgFor from a lazy loaded module, import CommonModule instead.`);
+                         'The `BrowserModule` has already been loaded. If you need access to ' +
+                         'common directives such as NgIf and NgFor from a lazy loaded module, ' +
```

Maybe we should update this error message now as we can get this error without lazy-loading situations? IMO dropping the "from a lazy loaded module" part should do the trick.

---

This error is now dropped in favor of a providers-based one, which doesn't reference lazy-loading at all.